mon-pix/tests/helpers/service-stubs.js

```diff
@@ -0,0 +1,103 @@
+import Service from '@ember/service';
+import sinon from 'sinon';
+
+/**
+ * Stub the session service.
+ *
+ * @param {Object} owner - The owner object.
+ * @param {Object} [session={}] - The session object.
+ * @param {boolean} [session.isAuthenticated=false] - Indicates if the user is authenticated.
+ * @param {boolean} [session.isAuthenticatedByGar=false] - Indicates if the user is authenticated by GAR.
+ * @param {string} [session.externalUserTokenFromGar='external-user-token'] - The external user token from GAR.
+ * @param {string} [session.userIdForAssociation='expected-user-id'] - The expected user ID from GAR.
+ * @param {string} [session.source='pix'] - The source of authentication.
+ * @returns {Service} The stubbed session service.
+ */
+export function stubSessionService(owner, session = {}) {
```

Comment:
Je n'en suis pas sûr mais je propose, je trouve que cela pourrait être plus clair en disant qu'on passe « des données de session », plutôt qu'une « session », sinon en tant que développeur je vais avoir l'impression qu'on me demande de passer « la vraie session ».

```suggestion
export function stubSessionService(owner, sessionData = {}) {
```